Tests/NewsServiceTests/NewsServiceTests.csproj

```diff
@@ -7,9 +7,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Moq" Version="4.13.1" />
```

Comment:
If we don't do mock anything, why we should to connect it?

Reply:
Yeah, you right 
fixed